test-runtime/index.html

```diff
@@ -1,111 +1,114 @@
 <!DOCTYPE html>
 <html>
```

Comment:
Why is this file changing so much?

Reply:
It seems like you have some kind of formatter that runs automatically. Please try not to make big changes like formatting changes in a pr. Keep the diff minimal.

Reply:
I've reset all of my formatting settings, which have resolved a good number of the formatting issues in the compiled examples. However, the "prettier" formatter used seems to be stubborn about specifically the spacing between `<html>` and `<head>`, which is changing the starting lines for all of the following code segments.

Reply:
I see. Ideally you would not include this in the pull request but I guess if prettier insists, we can accept this change now. Usually I'd like these formatting changes in a separate pull request.